disnake/invite.py

```diff
@@ -165,6 +167,11 @@ class PartialInviteGuild:
 
     verification_level: :class:`VerificationLevel`
         The partial guild's verification level.
+
+    welcome_screen: Optional[:class:`WelcomeScreen`]
+        The partial guild's welcome screen, if any.
+
+        .. versionadded:: 2.5
     premium_subscription_count: :class:`int`
```

Comment:
```suggestion
    verification_level: :class:`VerificationLevel`
        The partial guild's verification level.
    welcome_screen: Optional[:class:`WelcomeScreen`]
        The partial guild's welcome screen, if any.

        .. versionadded:: 2.5

    premium_subscription_count: :class:`int`
```

Reply:
https://github.com/DisnakeDev/disnake/pull/339/commits/7dff3cffc54ae17be2b7b47848deacdd602da658

Reply:
you forgot this one